### DISCUSSION TITLE
Repositories created without my knowledge

### DISCUSSION THREAD

**Discussion by waynezo** [community/community#136714] [Repositories] [answered]
### Select Topic Area

Question

### Body

I have used the "bypass-paywalls-chrome" extension, but I am not sure how I ended up with a repository.  I received this notice.

This repository is currently disabled due to a DMCA takedown notice. We have disabled public access to the repository. The notice has been [publicly posted](https://github.com/github/dmca/blob/master/2024/08/2024-08-09-news-media-alliance.md).

Also a repository was created for smartubenext. Was that because I posted a question? Please advise how these repositories are being created?

**Comment by davevad93** [comment, marked as answer]
Hi @waynezo ,

It's unusual for repositories to be created without your knowledge or consent, most likely your GitHub account has been compromised. I'm not familiar with the `bypass-paywalls-chrome` extension, but if you've started noticing strange things after using it, it's possible that it's the culprit and that it's malicious.

Check for any possible unauthorized activity by taking a look at your [security log](https://github.com/settings/security-log) in the settings; you can also contact support and request a security log [here](https://support.github.com/contact?flow=log_request&tags=rr-log-req).

**Reply by waynezo** [reply]
I suppose when I was looking for support I generated a pull request and I may have clicked the create repo or fork link.  I have not noticed any malicious activity in general or in the security log.  I destroyed the smartubenext repo using settings.  I put in a request to support to destroy "bypass-paywalls-chrome" repo.  The ticket has been open for 5 days.  Is that normal?

Remove bypass-paywalls-chrome repository #2951700
 Open	waynezo opened this ticket for waynezo 5 days ago · 2 comments

**Reply by davevad93** [reply]
Hey @waynezo, yeah GitHub Support may take a while to reply which is normal, however have you tried removing the 
`bypass-paywalls-chrome` forked repo by going to `Settings` => `Remove this repository` ?

**Reply by waynezo** [reply]
Thanks for your quick replies.  Yes I tried, I am denied access because of the DMCA takedown.  I am guessing I just have to wait for support to destroy/remove the "bypass-paywalls-chrome" repo.  I am grateful for your knowledge to help me understand this Github system.

**Reply by davevad93** [reply]
You're welcome @waynezo , have a nice day! 👍
